spec/rails_helper.rb

```diff
@@ -663,6 +663,7 @@ class SpecTimeoutError < StandardError
     end
 
     page.execute_script("if (typeof MessageBus !== 'undefined') { MessageBus.stop(); }")
+    Capybara.reset_session!
```

Comment:
This is required because we want to wait for all requests to finish before flushing redis